### DISCUSSION TITLE
what is the purpose of config password

### DISCUSSION THREAD

**Discussion by Ludo-code** [tiredofit/docker-openldap#62] [Q&A]
Hello, i'm using our docker openldap but i don't understand the utility of config password... 

I can't connect with it so i think i'm not able to use it... But where can i use it ?

**Comment by tiredofit** [comment]
CONFIG_PASS is for the cn=config DIT, essentially the backend configuration and schema setup. Most people normally don't need it.

https://www.zytrax.com/books/ldap/ch6/slapd-config.html#use

**Reply by Ludo-code** [reply]
Thanks you man.
